src/js/components/FieldCell.tsx

```diff
@@ -4,17 +4,31 @@ import classNames from "classnames"
 import {createCell} from "../brim/cell"
 import {zng} from "zealot"
 
-type Props = {field: zng.Field}
+type Props = {field: zng.Field; record: zng.Record}
 
-export default function FieldCell({field}: Props) {
+function getBackground(field, record) {
```

Comment:
This function adds the `alert-1-bg-color` class to event_type=alert cells in the viewer.

Reply:
It allows us to color the different alert severities with different colors: 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/3460638/103566504-f6e14d00-4e76-11eb-921a-e1a70d3aaf27.png">